app/forms/delivery_partner_form.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
```

Comment:
Seems to be a lot of logic in this form and well more than I would usually put in a form object. We may need to move some of it to a service object in future prs.

Reply:
Sounds good